crossbeam-queue/src/array_queue.rs

```diff
@@ -186,9 +188,7 @@ impl<T> ArrayQueue<T> {
                 ) {
                     Ok(_) => {
                         // Write the value into the slot and update the stamp.
-                        unsafe {
-                            slot.value.get().write(value);
-                        }
+                        unsafe { slot.value.get().write(MaybeUninit::new(value)) }
```

Comment:
I think it's clearer to add `;`, because it's a statement.

Reply:
Will do.